doc/plutus-core-spec/untyped-cek-machine.tex

```diff
@@ -134,6 +141,12 @@ \section{The CEK Machine}
                          s \return \VBuiltin{b}{\repetition{V}}{\eta} \enskip \text{if $\iota \in \QVar$}\\
 \inForceFrame{} \cons s & \return \VBuiltin{b}{\repetition{V}}{[\iota]}   &~\mapsto~&
                          \EvalCEK\,(s, b, \repetition{V}) \enskip \text{if $\iota \in \QVar$}\\
+\inConstrFrame{i}{\repetition{V}}{(M \cons \repetition{M}, \rho)} \cons s & \return V   &~\mapsto~&
+                         \inConstrFrame{i}{\repetition{V} \cons V}{(\repetition{M}, \rho)} \cons s;\rho \compute M \\
+\inConstrFrame{i}{\repetition{V}}{} \cons s & \return V   &~\mapsto~&
```

Comment:
It is implicitly assumed that the frame 
$$(\texttt{constr} ~ i  ~ \overline{V}~ \textunderscore ~ ) \quad = \quad (\texttt{constr} ~ i  ~ \overline{V}~ \textunderscore ~ ([  ],\rho) ) $$

Perhaps this should be stated explicitly?

Reply:
I should just write it the other way, yeah.